src/main/java/com/employeesService/EmployeesService/model/DepartmentEmployee.java

```diff
@@ -0,0 +1,83 @@
+package com.employeesService.EmployeesService.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+
+@Entity
+@Table(name = "dept_emp")
+public class DepartmentEmployee {
+
+    @EmbeddedId
+    private DepartmentEmployeeId id;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @MapsId("deptNo")
+    @JoinColumn(name = "dept_no")
+    @JsonIgnore
+    private Department department;
```

Comment:
```suggestion
    @JsonIgnore
    @MapsId("deptNo")
    @JoinColumn(name = "dept_no")
    @ManyToOne(fetch = FetchType.LAZY)
    private Department department;
```

Simply order the annotations a little better, so it better for eyes :D